github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
What is happening here?

Reply:
@kilemensi Fixed it

Reply:
You're adding `github` folder but we already have `.github` folder (with a `.` infront)